BUILDING.md

```diff
@@ -115,6 +115,7 @@ make generate
 >   * `no_btrfs`: A build tag disables building the Btrfs snapshot driver.
 >   * `no_devmapper`: A build tag disables building the device mapper snapshot driver.
 >   * `no_zfs`: A build tag disables building the ZFS snapshot driver.
+> * `no_dynamic_plugins`: A build tag disables dynamic plugins.
```

Comment:
Looks  like the formatting  is off here; would you be able to fix that up?  🙈

Reply:
I believe the formatting is correct, see [this comment](https://github.com/containerd/containerd/pull/11203#discussion_r1898963169)
(TLDR: immediately above is a list of tags related to snapshot, but if you check higher you can see other tags indented at the same level)

Reply:
Doh! My bad; I  was on the "changes" tab and missed that comment. Thanks!

Reply:
No worries !